tools/test/config/range_limits/lib1/mbed_lib.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "lib1",
+    "config": {
+        "config1": {
+            "help": "The default value should pass as it is in the list of accepted values",
+            "value": 5,
+            "accepted_values": "0, 5, 10"
```

Comment:
@kegilbert Any reason to prefer a comma-separated list over an array? An array seems more appropriate, but maybe there's a case I've missed.

Reply:
I mainly did this to move the error message formatting to the JSON file instead of massaging it in the error case. I can move this back to an array and format the string in the exception case again if that's more clear to users.